client/src/components/InboxUserProfilesGroup/InboxUserProfilesGroup.js

```diff
@@ -10,24 +10,40 @@ const InboxUserProfilesGroup = props => {
   // Default Class to apply to Component
   let classList = `InboxUserProfilesGroup`;
 
-  let profilePics = props.users.map((user, i) => {
-    if (i < 3) {
-      if (props.users.length >= 4 && i === 2) {
-        return (
-          <div className="InboxUserProfilesGroup__extra-users" key={i}>
-            <ExtraUsers to="/" users={props.users} />
-          </div>
-        );
-      }
-      return (
-        <div className="InboxUserProfilesGroup__profile-picture" key={i}>
-          <ProfilePicture type="extra-small" to={user.to} image={user.profilePic} />
+  return (
+    <div className={classList}>
+      <div className="InboxUserProfilesGroup__profile-picture" key={0}>
+        <ProfilePicture
+          type="extra-small"
+          to={props.users[0].to}
+          image={props.users[0].profilePic}
+        />
+      </div>
```

Comment:
Hey, this looks a lot better! but I think there may be a simpler way to do this with like `map` or something. Basically loop through the users passed in and once you get to three, just count how many are left and finish. It looks a lot better without the conditionals! I think you can make this more dry though

Reply:
I think I may have missed the point on the user profile group... I've simplified my code, but can't seem to get rid of one last conditional. If there are three users, should all three photos be displayed? or should 2 photos appear with a +1 next to them? Currently I have all three photos showing up, but could do 2 maps for the other way.